apps/website/src/pages/index.astro

```diff
@@ -435,4 +436,52 @@ const currentPage = new URL(Astro.request.url).pathname;
       }
     }
   </style>
+
+  <script>
```

Comment:
all of this could be moved to a separate file

Reply:
it can be a normal JS/TS file, or it can even be a .astro file.

if you go with .astro, you'll be able to move the styles into that file as well

Reply:
Moved the `<style>` and `<script>` into a separate file called `MoveOnVisible.astro`. I then included used that component in the `_global.astro` file.